src/tasks/base.py

```diff
@@ -41,9 +41,17 @@ def forward_with_gt(batch: Dict[str, Any]) -> Dict[str, torch.Tensor]:
         """Abstract forward method for training(with ground truth labels)."""
         pass
 
+    @abstractmethod
+    def no_weight_decay(self) -> List[str]:
```

Comment:
It should be not here but in `BaseModel`. Otherwise, a user has to setup no-weight-decay params in the task, but they are defined for each model custom which is overkill